docs/TROUBLESHOOTING.md

```diff
@@ -191,3 +192,45 @@ resource "google_artifact_registry_repository_iam_member" "python_reader" {
   member     = "serviceAccount:sa-dataflow-controller-reid@CONFIDENTIAL-DATA-PROJECT-ID.iam.gserviceaccount.com"
 }
 ```
+
+### `Bad syntax for dict argument` when deploying Dataflow Jobs using `gcloud` command
```

Comment:
are we considering removing this section, since this deviates from the recommended way?

Reply:
Yes, I removed this section.